cg/meta/compress/compress.py

```diff
@@ -112,6 +112,11 @@ def decompress_spring(self, sample_id: str) -> bool:
         for compression in compressions:
             if not self.crunchy_api.is_spring_decompression_possible(compression_obj=compression):
                 LOG.info(f"SPRING to FASTQ decompression not possible for {sample_id}")
+                if self.crunchy_api.is_compression_pending(compression):
```

Comment:
I think this catches pending decompressions as well, but need to double check and test.

Reply:
double-check that, otherwise I think this is the most straight-forward solution 👍

Reply:
<img width="609" alt="image" src="https://github.com/Clinical-Genomics/cg/assets/40887124/4b3f42c8-1a4f-4861-96da-7e991d351ea8">
Used here for decompression!